lms/djangoapps/certificates/api.py

```diff
@@ -231,6 +236,7 @@ def certificate_downloadable_status(student, course_key):
         'is_generating': True if current_status['status'] in [CertificateStatuses.generating,
                                                               CertificateStatuses.error] else False,
         'is_unverified': True if current_status['status'] == CertificateStatuses.unverified else False,
+        'is_dateunavailable': True if current_status['status'] == CertificateStatuses.date_unavailable else False,
```

Comment:
This is just a nitpick, but I'd think about adding an underscore in this variable name, so `is_date_unavailable`.